src/react/features/buttons/actions.js

```diff
@@ -1,25 +1,21 @@
-import { ADD_BUTTONS, REMOVE_BUTTONS, SET_OPEN, SET_ACTIVE, SET_INACTIVE } from './actionsTypes'
+import { ADD_BUTTONS, REMOVE_BUTTONS, SET_OPEN, SET_ACTIVE } from './actionsTypes'
 
-export const addButtons = (buttons) => ({
+export const addButtons = (addedButtons) => ({
   type: ADD_BUTTONS,
-  payload: { buttons },
+  payload: { addedButtons },
 })
 
-export const removeButtons = (buttons) => ({
+export const removeButtons = (removedButtons) => ({
   type: REMOVE_BUTTONS,
-  payload: { buttons },
+  payload: { removedButtons },
 })
 
 export const setOpen = (open) => ({
   type: SET_OPEN,
   payload: { open },
 })
 
-export const setActive = (active) => ({
+export const setActive = (buttonId) => ({
   type: SET_ACTIVE,
-  payload: { active },
-})
-
-export const setInactive = () => ({
```

Comment:
One reduc style guide:
"However, we recommend trying to treat actions more as "describing events that occurred", rather than "setters". Treating actions as "events" generally leads to more meaningful action names, fewer total actions being dispatched, and a more meaningful action log history."

According to this style guide, we should go to other direction and change setOpen to openButtons and add closeButtons; and rename setActive to activeButton and inactivateButton.

Reply:
Thanks for the good quote from style guide and good suggestion for renaming!

Renaming done: 
```
setOpen -> openButtons
setActive -> activateButton
setInactive -> inactivateButton
```

closeButtons wasn't added because it is not used anywhere... Everytime we close buttons we want also to inactivateButton so it doesn't make sense to add it if it is not used...